dspace/config/log4j2.xml

```diff
@@ -64,6 +64,18 @@
             </DefaultRolloverStrategy>
             -->
         </Appender>
+
+        <!-- A3 is for tracking file downloads  -->
+        <Appender name='A3'
+                  filePattern="${log.dir}/file_downloads.log-%d{yyyy-MM-dd}"
```

Comment:
file vs bitstream?

Reply:
OK called that file in one conversation as `file_downloads` and I like it. When the user downloads some license.txt it should put this info into log file and it is not bitstream. I think `file` is more appropriate than bitstream.